rpc/connection.go

```diff
@@ -599,7 +607,8 @@ func (c *Connection) IsConnected() bool {
 func (c *Connection) getReconnectChanLocked() (
 	reconnectChan chan struct{}, disconnectStatus DisconnectStatus,
 	reconnectErrPtr *error) {
-	c.log.Debug("Connection: getReconnectChan")
+	c.log.Info("Connection: %s",
```

Comment:
Didn't we make these `Info` before, and then change it back to `Debug` because it was showing up in users' terminals?

Reply:
Yup; in https://github.com/keybase/client/pull/10361 we are changing those that rely on INFO to print to terminal to use the dumb UI printer, and change the terminal log level from INFO to NOTICE. So this won't leak to CLI anymore.